### PR TITLE
Rename `datagas` to `blobgas` - eip 4844 pr 7354

### PR DESCRIPTION
**Pull request by acolytec3**
This implements [EIP 4844 PR 7354](https://github.com/ethereum/EIPs/pull/7354) to rename `dataGas` to `blobGas` in all instances

### CODE REVIEW

File: packages/block/src/block.ts [hunk 1]
```diff
@@ -559,48 +559,48 @@ export class Block {
...
+      if (this.header.blobGasUsed !== blobGasUsed) {
         throw new Error(
-          `block dataGasUsed mismatch: have ${this.header.dataGasUsed}, want ${dataGasUsed}`
+          `block blobGasUsed mismatch: have ${this.header.blobGasUsed}, want ${blobGasUsed}`
         )
       }
 
-      const expectedExcessDataGas = parentHeader.calcNextExcessDataGas()
-      if (this.header.excessDataGas !== expectedExcessDataGas) {
+      const expectedExcessblobGas = parentHeader.calcNextExcessblobGas()
```
Comment (g11tech) [inline packages/block/src/block.ts:600]:
```suggestion
      const expectedExcessBlobGas = parentHeader.calcNextExcessBlobGas()
```
just to maintain consistency

File: packages/block/src/block.ts [hunk 2]
```diff
@@ -559,48 +559,48 @@ export class Block {
...
+      if (this.header.excessblobGas !== expectedExcessblobGas) {
```
Comment (g11tech) [inline packages/block/src/block.ts:601]:
```suggestion
      if (this.header.excessBlobGas !== expectedExcessblobGas) {
```
for direct mapping with the snake to camel conversion consistency

**Review by g11tech — DISMISSED** [review]
added a commit which PR seems to be on some lag to show, but should work post it

approving the changes tentatively

**Review by g11tech — APPROVED** [review]

**Review by g11tech — APPROVED** [review]
lgtm

**Comment by holgerd77** [comment]
Just to highlight the new `type: spec updates` label we have for some time now in the context of this PR, I think this is helpful for PR classification, since lately (and especially with 4844) some PRs with some changes regarding spec updates have gotten pretty common/frequent.

**Pull request merged**